doc/lua_api.txt

```diff
@@ -845,6 +845,131 @@ from the available ones of the following files:
 * (...)
 * `foomod_foosound.9.ogg`
 
+
+`SimpleSoundSpec`
+-----------------
+
+Specifies a sound name, gain (=volume), pitch and fade.
+This is either a string or a table.
+
+In string form, you just specify the sound name or
+the empty string for no sound.
+
+Table form has the following fields:
+
+{
+* `name`: Sound name
+* `gain`: Volume (`1.0` = 100%)
+* `pitch`: Pitch (`1.0` = 100%)
+* `fade`: Fade (`0.0` = no fade) 
+}
+
+`gain` and `pitch` are optional and default to `1.0`.
+`fade` is optional and defaults to `0.0`, which means no fade is applied.
+
+`gain` is capped at 1. Any value above it will be heard equally loud.
+
+`fade` values close to 0 make the sound almost inaudible, as opposed to 0:
+
+The `fade`, `gain` parameters correspond to the 'step', 'gain' parameters of 
+`minetest.sound_fade`, with the exception of the default value 'fade=0.0', 
+which disables any fading effect. Refer to `minetest.sound_fade()` documentation 
+in the section 'minetest' namespace reference / Sounds.
+
+
+
+Examples:
+
+* `""`: No sound
+* `{}`: No sound
+* `"default_place_node"`: Play e.g. `default_place_node.ogg`
+* `{name = "default_place_node"}`: Same as above
+* `{name = "default_place_node", gain = 0.5}`: 50% volume
+* `{name = "default_place_node", gain = 0.9, pitch = 1.1}`: 90% volume, 110% pitch
+
+
+`Sound Parameters`
+------------------
+Basic sound functionality as used within node definitions for placing, breaking, 
+digging, walking, swimming etc. only uses the SimpleSoundSpec.
+The minetest.sound_play() function takes an optional second table of parameters in 
+addition to SimpleSoundSpec. 
+
+This table has the following parameters, all of which are optional, same as the 
+table itself (when omitted, default values are used for the values that have them):
+
+{
```

Comment:
Your formatting here is severely broken - you only semi-converted the Lua code snippet to Markdown...